### ISSUE TITLE
W0511: standalone 'XXX' does not cause warning

### ISSUE THREAD

**Issue by regisz** [PyCQA/pylint#2489]
### Steps to reproduce
1. write `# XXX` into code

If I write anything after XXX, then warning can be seen.

### Current behavior
no W0511 warning

### Expected behavior
standalone XXX causes warning in comment

### pylint --version output
I use pymode, and it uses this pylint hash: 66cb32187c040f82dd067bc0d226b2f105bf6c38.
I do not use pylint directly.

**Comment by PCManticore** [comment]
I cannot reproduce this problem using the following:
```
# XXX
```
Getting the following output:
```
a.py:1:2: W0511: XXX (fixme)
a.py:1:0: C0111: Missing module docstring (missing-docstring)
```
Is your reproduction example different than this one? I'm closing it for now, but if you can still reproduce it, let's reopen it with a more complete example.

**Comment by regisz** [comment]
@PCManticore I tried it again, and the problem exists for me yet.

**Comment by PCManticore** [comment]
I honestly cannot reproduce this with latest `pylint`, installed from the `master` branch:
```
⋊> ~/p/p/playground pylint a.py                                                                                   10:04:05
************* Module a
a.py:1:2: W0511: XXX a (fixme)
⋊> ~/p/p/playground pylint a.py                                                                                   10:04:21
************* Module a
a.py:1:2: W0511: XXX (fixme)
⋊> ~/p/p/playground pylint --version                                                                              10:04:23
pylint 2.2.0-dev
astroid 2.1.0-dev
Python 3.6.5 (default, May  7 2018, 10:07:07)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)]
```

**Comment by regisz** [comment]
@PCManticore I wrote, I use 66cb321. Maybe this was a real error, but have it already been fixed?
